hapi-utils/src/main/java/com/hedera/services/contracts/ParsingConstants.java

```diff
@@ -96,6 +96,18 @@ private ParsingConstants() {
             TupleType.parse(RESPONSE_STATUS_AT_BEGINNING + TOKEN_INFO + ")");
     public static final TupleType getNonFungibleTokenInfoType =
             TupleType.parse(RESPONSE_STATUS_AT_BEGINNING + NON_FUNGIBLE_TOKEN_INFO + ")");
+    public static final TupleType tokenGetCustomFees =
+            TupleType.parse(
+                    RESPONSE_STATUS_AT_BEGINNING
+                            + FIXED_FEE
+                            + ARRAY_BRACKETS
+                            + ","
+                            + FRACTIONAL_FEE
+                            + ARRAY_BRACKETS
+                            + ","
+                            + ROYALTY_FEE
+                            + ARRAY_BRACKETS
+                            + ")");
```

Comment:
Wouldn't StringBuilder `append` chain be preferable to the `+` operator with such a high number of concatenations?

Reply:
Yes, it would be a cleaner approach. I suggest, however, to refactor all these concatenations at once in a future PR, in order to follow consistent logic.